bin/sparkling-env.sh

```diff
@@ -59,7 +59,11 @@ function getJarsArg() {
 
 
 function checkPythonPackages() {
-    packages=$(pip list --format=freeze)
+    PYTHON_BINARY=python
+    if [ -n "$PYSPARK_PYTHON" ]; then
+      PYTHON_BINARY=$PYSPARK_PYTHON
+    fi
+    packages=$($PYTHON_BINARY -m pip list --format=freeze)
```

Comment:
Is this this going to work with Python 3 (commands: `python3` and `pip3`)?

Reply:
It definitely does not break what is there right now. Default is still `python` and if PYSPARK_PYTHON points to python3 then all good as well